chpl/chpl-service/src/main/java/gov/healthit/chpl/service/CertificationCriterionService.java

```diff
@@ -39,14 +40,16 @@ public CertificationCriterionService(CertificationCriterionDAO certificationCrit
 
     @PostConstruct
     public void postConstruct() {
-        criteriaMap = certificationCriterionDAO.findAll().stream()
+        criteriaByIdMap = certificationCriterionDAO.findAll().stream()
                 .map(criterion -> new CertificationCriterion(criterion))
                 .collect(Collectors.toMap(CertificationCriterion::getId, cc -> cc));
+        criteriaByIdMap.values().stream()
+            .forEach(criterion -> insertCriterionIntoNumberMap(criterion));
```

Comment:
Could this line be inserted in-between the `map` and `collect` lines above?

Reply:
This second section is filling in a totally different map so I think it would be weird to move it into the above section that's assigning the value to the criteriaIdMap. I rearranged it somewhat in a way that I think is more clear.

Reply:
I missed that it was a different map